vector/src/main/res/layout/fragment_invites.xml

```diff
@@ -20,17 +20,24 @@
 
     </com.google.android.material.appbar.AppBarLayout>
 
-    <androidx.recyclerview.widget.RecyclerView
-        android:id="@+id/invites_recycler"
+    <im.vector.app.core.platform.StateView
+        android:id="@+id/invites_state_view"
         android:layout_width="0dp"
         android:layout_height="0dp"
-        android:fastScrollEnabled="true"
-        android:overScrollMode="always"
-        android:scrollbars="vertical"
-        app:layout_behavior="@string/appbar_scrolling_view_behavior"
```

Comment:
To double check, is it OK to remove this layout_behavior?

Reply:
yes, we don't need it for state view